crates/goose/src/developer.rs

```diff
@@ -234,7 +234,8 @@ impl DeveloperSystem {
 
         let instructions = formatdoc! {r#"
             The developer system is loaded in the directory listed below.
-            You can use the shell tool to run any command that would work on the relevant operating system.
+            You can use the shell tool to run any command that would work on the relevant operating system, 
+            such as api calls, web access/crawling and other things a developer would do from a command line.
```

Comment:
have you noticed any degraded performance for the other usual tasks?

Reply:
not enough data to know which give me pause. What I am thinking of doing instead is have a seperate system for "non developers" which would need to have a prompt like this (usually developers don't need to do this).